test/commands/atomicity_test.cpp

```diff
@@ -153,15 +153,15 @@ TEST_F(AtomicityTestFixture, AddVTB) {
   // endorsed
   auto vbk5 = popminer->vbk().getBestChain().tip()->getAncestor(5);
   ASSERT_TRUE(vbk5);
-  auto vbkpoptx1 = popminer->createVbkPopTxEndorsingVbkBlock(
-      vbk5->getHeader(), getLastKnownBtcBlock());
-  auto vbkpoptx2 = popminer->createVbkPopTxEndorsingVbkBlock(
-      vbk5->getHeader(), getLastKnownBtcBlock());
+  auto vbkpoptx1 =
+      popminer->endorseVbkBlock(vbk5->getHeader(), getLastKnownBtcBlock());
```

Comment:
I think `endorseVbkBlock` is less descriptive name than `createVbkPopTxEndorsingVbkBlock`. I'd revert back to old name

Reply:
Yeah, that's what I'm talking about. I don't know how to name function clearly, which:
- create BtcTx with published VbkBlock
- mine just created BtcTx
- create VbkPopTx with containing BtcTx and BtcBlock of proof.
I thought that `createVbkPopTxEndorsingVbkBlock` is also not clear enough, because it doesn't only create VbkPopTx, but perform a half of endorsement by mining in BTC as well. We also have atomic `createVbkPopTxEndorsingVbkBlock` function which only creates VbkPopTx without any side effects.
Maybe you have any alternative names? Or think `createVbkPopTxEndorsingVbkBlock` is OK for now?

Reply:
Renamed for now.
If we'll come with more appropriate name, we'd change it.